traitsui/testing/tester/qt4/implementation/range_editor.py

```diff
@@ -0,0 +1,87 @@
+#  Copyright (c) 2005-2020, Enthought, Inc.
+#  All rights reserved.
+#
+#  This software is provided without warranty under the terms of the BSD
+#  license included in LICENSE.txt and may be redistributed only
+#  under the conditions described in the aforementioned license.  The license
+#  is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+#  Thanks for using Enthought open source!
+#
+
+from traitsui.qt4.range_editor import (
+    LargeRangeSliderEditor,
+    LogRangeSliderEditor,
+    RangeTextEditor,
+    SimpleSliderEditor,
+)
+
+from traitsui.testing.tester import locator
+from traitsui.testing.tester.qt4.located_object_handlers import LocatedTextbox
+
+
+def resolve_location_slider(wrapper, location):
+    """ Solver from a UIWrapper wrapped Range Editor to a LocatedTextbox
+    containing the textbox of interest
+
+    If there are any conflicts, an error will occur.
+
+    Parameters
+    ----------
+    wrapper : UIWrapper
+        Wrapper containing the Range Editor target.
+    location : locator.WidgetType.textbox
+        The location we are looking to resolve.
+    """
+    if location == locator.WidgetType.textbox:
+        return LocatedTextbox(textbox=wrapper.target.control.text)
+
+    raise NotImplementedError()
```

Comment:
That makes sense, I think it doesn't hurt to include the `NotImplementedError`.  Only having the check for a slider seems quite specific though.  TBH I'm not entirely sure what all the widget types would be - but we could simply do

```if location in [locator.WidgetType.slider, locator.WidgetType.tabbar, etc.]:
    raise NotImplementedError(f"Logic for interacting with the {location} has not been implemented.")```

Reply:
I currently just made it a singleton list `[locator.WidgetType.slider]` as I wasn't sure what all widgets would be possible.  This can easily be added to.